### DISCUSSION TITLE
Rate adjust >2x

### DISCUSSION THREAD

**Discussion by TextAdventurer12** [ppy/osu#28851] [Ideas]
The 2x limiter is pretty arbitrary, and some recent [scores](https://www.reddit.com/r/osugame/comments/1e0ksdo/mrekk_set_it_off_horrible_kids_kukis_extra_dt/) have shown that people are capable of playing greater than that limit, and will likely want to when rate adjust awards PP. I don't see the harm in extending to 2.5, or even 3x, although precision of the slider would become a concern beyond a certain cap. Maybe an extended limits toggle could be added, similar to how difficulty adjust works.

**Comment by behsn** [comment]
I feel like the "harm" of this is actually to your ears, even at 2x most songs are just unlistenable lol

**Reply by Stoppedpuma** [reply]
Most people even using 2x are most likely doing it for the difficulty, not so much listening to the song.
